### PR TITLE
Added Unit test fo dassert_strictly_increasing_index()

### PR DESCRIPTION
**Pull request by mayank922**
Task #999

### CODE REVIEW

**Comment by mayank922** [comment]
Following are the test cases I wrote:
1.  Check that unique and monotonically increasing index passes the assert
2.  Check that Duplicate but monotonically increasing index raises the assert
3. Check that assert is raised if the index is unique but not monotonically increasing

I have considered the following corner cases but they are already satisfied in the code by other functions only if those are used with the input:

1. Empty DataFrame
2. The Dataframe doesn't have index as datetime

**Comment by mayank922** [comment]
Hi @samarth9008 

Awaiting your review.

File: helpers/test/test_hpandas.py [hunk 1]
```diff
@@ -3483,6 +3483,79 @@ def test3(self) -> None:
 # #############################################################################
 
 
+class Test_dassert_strictly_increasing_index(hunitest.TestCase):
+    def test1(self) -> None:
+        """
+        Check that unique and monotonically increasing index passes the assert
```
Comment (DanilYachmenev) [inline helpers/test/test_hpandas.py:3489]: Finish all docstrings and comments with dots

File: helpers/test/test_hpandas.py [hunk 2]
```diff
@@ -3483,6 +3483,79 @@ def test3(self) -> None:
...
+            pd.Timestamp("2000-01-01 9:04"),
+        ]
+        values = [0, 0, 0, 0]
+        df = pd.DataFrame(values, index=idx)
+        # Run.
+        hpandas.dassert_strictly_increasing_index(df)
+
+    def test2(self) -> None:
+        """
+        Check that Duplicate but monotonically increasing index raises the assert
```
Comment (DanilYachmenev) [inline helpers/test/test_hpandas.py:3505]: -> `Check that an assert is raised for an increasing index with duplicates.`

File: helpers/test/test_hpandas.py [hunk 3]
```diff
@@ -3483,6 +3483,79 @@ def test3(self) -> None:
...
+        cond=False
+        Duplicated rows are:
+                            0
+        2000-01-01 09:01:00  0
+        2000-01-01 09:01:00  0"""
+        self.assert_equal(act, exp, fuzzy_match=True)       
+
+    def test3(self) -> None:
+        """
+        Check that assert is raised if index is unique but not monotonically increasing
```
Comment (DanilYachmenev) [inline helpers/test/test_hpandas.py:3531]: -> `Check that an assert is raised for a not monotonically increasing index.`

**Review by DanilYachmenev — COMMENTED** [review]
some docstring nits but good job overall

**Comment by mayank922** [comment]
Hi @DanilYachmenev 

I have made the changes.

That rephrased version you suggested sounds better.

File: helpers/test/test_hpandas.py [hunk 4]
```diff
@@ -3891,25 +3964,19 @@ def get_multiindex_df(
...
-                2022-01-01 21:10:00+00:00   1.303778    -0.288235
-        index2  2022-01-01 21:00:00+00:00   1.237079    1.168012
-                2022-01-01 21:10:00+00:00   1.333692    1.708455
-        ```
+        ```                                     column1     column2
+        index   timestamp index1  2022-01-01 21:00:00+00:00   -0.122140
+        -1.949431         2022-01-01 21:10:00+00:00   1.303778
+        -0.288235 index2  2022-01-01 21:00:00+00:00   1.237079
+        1.168012         2022-01-01 21:10:00+00:00   1.333692
+        1.708455 ```
```
Comment (DanilYachmenev) [inline helpers/test/test_hpandas.py:3972]: linter fixes it automatically but it is a corner case for it so lets revert these changes back to master
Keep only the ones that you've added

**Review by DanilYachmenev — COMMENTED** [review]
linter stabbed you, but its ok
it is important to lint your code in any case so gj doing it

apart from that all good

File: helpers/test/test_hpandas.py (hunk 4 above)
Comment (mayank922) [reply helpers/test/test_hpandas.py]: That's what confused me like I didn't make these changes.

I have reverted it to the original

Comment (DanilYachmenev) [reply helpers/test/test_hpandas.py]: linter might do such changes so a good practice is to briefly double-check the edited files to make sure it has not spoiled anything

File: helpers/test/test_hpandas.py [hunk 5]
```diff
@@ -3911,6 +3984,7 @@ def get_multiindex_df(
                 string2     1.333692    1.708455
         ```
         """
+
```
Comment (DanilYachmenev) [inline helpers/test/test_hpandas.py:3987]: revert this line too pls
